src/patch-events.js

```diff
@@ -275,6 +275,10 @@ export function addEventListener(type, fn, optionsOrCapture) {
           return;
         }
       }
+      if(fn.handleEvent) {
```

Comment:
Hmm, this seems to work counter to my reading of the specification: https://dom.spec.whatwg.org/#concept-event-listener-inner-invoke.

But this JSBin example shows that you are correct: http://jsbin.com/gagoyof/edit?html,console

Regardless, my first comment still stands. A function with `handleEvent` property still executes the function body: http://jsbin.com/jodolar/edit?html,console